tests/xso/test_model.py

```diff
@@ -67,6 +68,18 @@ def make_instance_mock(mapping={}):
     return instance
 
 
+def unparse_to_node(xso, parent):
```

Comment:
Do you remove `XSO.unparse_to_node` or just replace its uses in the tests for future removal of `XSO.unparse_to_node` (I can't find the diff, where the XSO method is removed).

Reply:
Only the former.